frontend/appflowy_web_app/src/utils/url.ts

```diff
@@ -7,7 +7,7 @@ export const downloadPage = 'https://appflowy.io/download';
 
 export const openAppFlowySchema = 'appflowy-flutter://';
 
-export const iosDownloadLink = 'https://testflight.apple.com/join/6CexvkDz';
+export const iosDownloadLink = 'https://apps.apple.com/us/app/appflowy/id6457261352';
```

Comment:
my fault. remove the 'us'. :)

```suggestion
export const iosDownloadLink = 'https://apps.apple.com/app/appflowy/id6457261352';
```